### PR TITLE
Add log messages to ppm changes: MB-1399

### PR DESCRIPTION
**Pull request by lynzt**
## Description

It's hard to track down issues affecting ppm moves. This code adds logs for when ppms are created/updated/patched. The logs also have a `moveLocator` which makes searching cloudWatch a bit easier...

## Reviewer Notes

`logDetails` is general and could be used to log other actions if we want. Didn't move it to another file, but could... idk.

## Setup

`make server_run`
`make client_run`
Create a move as a SM - I've been using `needs@orde.rs` as a starting point. Watch the server logs for PPM specific output (`Create PPM`/`Patch PPM`/`Update PPM`)

On the office app, you can the move details and you should also see PPM log output in the server logs.

## Code Review Verification Steps

* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-1399) for this change

### CODE REVIEW

**Comment by lynzt** [comment]
@chrisgilmerproj that makes sense... I was trying something fancy that could be used in more places and I was going to add things to bypass fields that are PII, but that would rely on some knowing that this is logging those values. It was also a place to try my hand and writing a reflect, but you're right... could be a security concern.

I'll go back to my more brittle but safer logs ⚡️

File: pkg/handlers/internalapi/personally_procured_move.go [hunk 1]
```diff
@@ -554,3 +570,45 @@ func (h RequestPPMExpenseSummaryHandler) Handle(params ppmop.RequestPPMExpenseSu
 
 	return ppmop.NewRequestPPMExpenseSummaryOK().WithPayload(&expenseSummaryPayload)
 }
+
+func logDetails(logger Logger, move models.Move, ppm models.PersonallyProcuredMove, msg string) {
```
Comment (chrisgilmerproj) [inline pkg/handlers/internalapi/personally_procured_move.go]: I have a feeling that this is going to flood the logs. Would it be better to instead just have a log line that says what the action is and the move locator id? And then we could look up the data for the move in that way? Or is this a stop block while we wait for the auditing in the DB to be available? I guess I'm driving at this being temporary and not permanent.

Also, if you do go forward with this, can you add a comment that says "Absolutely NO PII in this function?" In fact, I'm even worried that `pickupPostalCode` and `destinationPostalCode` might be too much info here.

Comment (lynzt) [reply pkg/handlers/internalapi/personally_procured_move.go]: This logging is due to trying to figure out what happened in the ppm state when Linda get's a message about a possible problem. She wants to see the info the SM and/or office enters and when. Basically, there are times when we see moves in production and we can't replicate the data on staging and could help w/ that...

Comment (chrisgilmerproj) [reply pkg/handlers/internalapi/personally_procured_move.go]: I think we talked earlier about this on Slack. I'll let you fill in Linda on the details about that conversation.

**Comment by lynzt** [comment]
closing this... will re-open if needed but as of now, this isn't needed...

**Pull request closed without merging**